storage/gcsemu/gcsemu.go

```diff
@@ -163,6 +179,8 @@ func (g *GcsEmu) Handler(w http.ResponseWriter, r *http.Request) {
 			g.handleGcsCopy(ctx, baseUrl, w, bucket, object)
 		} else if r.Form.Get("upload_id") != "" {
 			g.handleGcsNewObjectResume(ctx, baseUrl, w, r, r.Form.Get("upload_id"))
+		} else if strings.Contains(r.URL.Path, "/batch/") {
+			g.handleGcsBatch(ctx, baseUrl, w, r, bucket, object)
```

Comment:
Rather than wire this in here, I kind of think we should just export `(g *GcsEmu) BatchHandler` as a top-level HTTP handler, totally separate from the root handler, and just explicitly register `/batch/storage/v1` as a top-level hander.

Reply:
done